README.md

```diff
@@ -6,6 +6,43 @@ Requires Krita 5.1
 ## Usage example
 [target.webm](https://user-images.githubusercontent.com/112324253/188291339-9d146a9a-ba9f-4671-9bd8-c8b55fd48ba6.webm)
 
+[Detailed feature showcase with images, art by Greg Rutkowski](https://github.com/AUTOMATIC1111/stable-diffusion-webui-feature-showcase)

```

Comment:
Generally there are many things that just not in the Krita plugin. It's a bit misleading. Like there is no X/Y plot, no colab support, no variations. I would prefer not to copy README from upstream, and just link upstream instead. 

The only thing worth change would be installation instructions. I'll update it anyway soon.